CentralUnit/Communication/XbeeNetwork.cs

```diff
@@ -238,23 +238,27 @@ private XBeePacket CheckPacketComplete(byte[] buffer, int position)
             {
                 ushort packetLength = (ushort)(((ushort)buffer[1]) << 8);
                 packetLength |= (ushort)buffer[2];
+                packetLength += (ushort)buffer.Count(bt => bt == (byte)SpecialByte.ESCAPE_BYTE);
+                packetLength += 4;
```

Comment:
i would like a comment to say where this 4 is coming from

Reply:
or maybe just in general how the packet lengths works, since its a 3 line calculation